website/docs/artist_getting_started.md

```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
 
+**openpype_gui.exe** located in the OpenPype folder. This executable being suitable **for artists**.
 
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
+or alternatively by
 
-<TabItem value="win">
+**openpype_console.exe** which is more suitable for **TDs/Admin** for debugging and error reporting. This one runs with opened console window where all the necessary info will appear during user's work session.
 
-WIP - Windows instructions once installers are finished
+> By seeing the "OP" icon in the OS tray user can easily tell OpenPype already running.
 
-</TabItem>
-<TabItem value="linux">
 
-WIP - Linux instructions once installers are finished
 
-</TabItem>
-<TabItem value="mac">
+## First Launch
 
-WIP - Mac instructions once installers are finished
 
-</TabItem>
-</Tabs>
+When you first start OpenPype, you will be asked to fill in some basic informations.
 
+### MongoDB
 
-## First Launch
+In most cases that will only be your studio MongoDB Address.
+It's a URL that you should have received from your Studio admin and most often will look like this 
 
+`mongodb://username:passwword@mongo.mystudiodomain.com:12345`
 
-When you first start OpenPype, you will be asked to give it some basic information.
-### MongoDB
+ or
 
-In most cases that will only be your studio MongoDB Address.
+ `mongodb://192.168.100.15:27071`
 
-It is a URL that you should receive from you studio and most often will look like this `mongodb://username:passwword@mongo.mystudiodomain.com:12345` or  `mongodb://192.168.100.15:27071`, it really depends on your studio setup. When OpenPype Igniter
+it really depends on your studio setup. When OpenPype Igniter
 asks for it, just put it in the corresponding text field and press `install` button.
 
 ### OpenPype Version Repository
 
-Sometimes your studio might also ask you to fill in the path to it's version
+Sometimes your Studio might also ask you to fill in the path to it's version
 repository. This is a location where OpenPype will be looking for when checking
 if it's up to date and where updates are installed from automatically. 
 
-This pat is usually taken from the database directly, so you shouldn't need it. 
+This path is usually taken from the database directly, so you shouldn't need it. 
 
 
 ## Updates
 
-If you're connected to your studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start, it will go through a quick update installation process, even though you might have just installed it. 
+If you're connected to your Studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start, it will go through a quick update installation process, even though you might have just installed it. 
```

Comment:
```suggestion
If you're connected to your Studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start it can go through a quick update installation process, even though you might have just installed it. 
```